lib/internal/per_context.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+// node::NewContext calls this script
+
+(function(global) {
+  // https://github.com/nodejs/node/issues/14909
+  delete global.Intl.v8BreakIterator;
+
+  // https://github.com/nodejs/node/issues/21219
+  // Adds Atomics.notify and warns on first usage of Atomics.wake
+
+  const AtomicsWake = global.Atomics.wake;
+  const ReflectApply = global.Reflect.apply;
+
+  // wrap for function.name
+  function notify(...args) {
+    return ReflectApply(AtomicsWake, this, args);
+  }
+
+  const warning = 'Atomics.wake will be removed in a future version, ' +
+    'use Atomics.notify instead.';
+
+  let wakeWarned = false;
+  function wake(...args) {
+    if (!wakeWarned) {
+      wakeWarned = true;
+
+      if (global.process !== undefined) {
+        global.process.emitWarning(warning, 'Atomics');
+      } else {
+        global.console.error(`Atomics: ${warning}`);
+      }
+    }
```

Comment:
I’m not sure, is there any reason why we wouldn’t want to do a normal deprecation cycle for these (i.e. make this PR semver-major)?

Reply:
I wasn't quite sure where to document it and like @rwaldon said it's not very widely used.

Reply:
should this still be considered semver major?